ipfsApi/client.py

```diff
@@ -201,16 +202,21 @@ def add(self, files, recursive=False, **kwargs):
         return self._add.request(self._client, (), files,
                                  recursive=recursive, **kwargs)
 
-    def get(self, multihash, **kwargs):
+    def get(self, multihash, output_path='.', **kwargs):
         """Downloads a file, or directory of files from IPFS.
 
-        Files are placed in the current working directory.
+        Files are placed in the current working directory if
+        output_path is not specified. Otherwise, they are
+        moved to output_path.
 
         Keyword arguments:
         multihash -- unique checksum used to identify IPFS resources
+        output_path -- where to put the downloaded file
         kwargs -- additional named arguments
         """
-        return self._get.request(self._client, multihash, **kwargs)
+        self._get.request(self._client, multihash, **kwargs)
+        if output_path is not '.':
+            shutil.move(multihash, output_path)
```

Comment:
Is there really no better way to do this than to call `shutil.move`? What if the output file is very big and the output path is on a different file system? What if the current working directory isn't writable?